### ISSUE TITLE
Unable to delete a file using Os.X 10.15

### ISSUE THREAD

**Issue by jonesberg** [fman-community/fman#716]
When I attempt to delete a file using fman, the file disappear for a few seconds from the window, then re-appears at the exact spot where it was.

It also doesn't appear into the trash when I use fman. Using another file manager works fine.

It seems to be related to permissions, as it works when I launch fman from the command line (using iTerm2). It asked me if I wanted iTerm2 to control Finder.app and then deleted the file properly. (I see it appear in the Trash folder of the finder).

I deleted and re-installed fman just to be sure, without any success.

**Comment by scuq** [comment]
same here with 10.15.2, even with "Full Disk Access" rights for fman.

my workaround is a remap of the move to thrash keys to delete_permanently

```
$ more Key\ Bindings.json | grep delete
	{ "keys": ["F8"], "command": "delete_permanently" },
	{ "keys": ["Delete"], "command": "delete_permanently" },
	{ "keys": ["Shift+Delete"], "command": "delete_permanently" },
```

**Comment by marvelph** [comment]
If you are using a Finder internally, macOS Catalina may need to have automation permission.

**Comment by raguay** [comment]
Or, if you turn finder off (yes, you can do that), then it will not delete
the items. That is the reason for my permanent delete plugin.

**Comment by marvelph** [comment]
I found a workaround.
Run "Get Info" once on any file and give Finder automation permission.
Then the move to the trash works.

**Comment by EYYCHEEV** [comment]
I have the same problem, please help

**Comment by EYYCHEEV** [comment]
> I found a workaround.
> Run "Get Info" once on any file and give Finder automation permission.
> Then the move to the trash works.

@marvelph This works, thanks.

**Comment by mherrmann** [comment]
I just released [fman 1.7.1](https://fman.io/blog/improvements-for-macos-catalina/) that addresses this problem. Sorry for the trouble.

**Comment by bluj** [comment]
Just wanted to let you know, that I had to find this workaround with "get info", because deleting didn't work.

System Info:
MacOS 10.15.4
I had fman 1.4.x installed, but never used it before (maybe once for 10 seconds after I had initially installed it over a year ago I guess). Now I wanted to try it out. After second restart it got updated to 1.7.2 automatically as intended. And deleting still didn't work until I did the little "get info" workaround that led me to the window to give "automation" permissions.
